docs/metadata/concepts.md

```diff
@@ -210,9 +210,9 @@ registration [template sheet](templates/sample-metadata-template.xlsx).
 #### Batch
 
 !!! info
-    A batch can be defined as a group of samples processed together under the same
-    experimental conditions.
-    <br/>  This is done to document and minimize technical variation.
+A batch can be defined as a group of samples processed together under the same
+experimental conditions.
+<br/>  This is done to document and minimize technical variation.
```

Comment:
This reintroduces the indentation issue. Autoformatting in markdown is horrible 😢